src/parsers/FibonacciDecoder.ts

```diff
@@ -60,4 +60,18 @@ export default class FibonacciDecoder {
 
     throw Error('Did not find termination bit');
   }
+
+  findLargestValue() {
+    const { numberOfValuesRead, currentValue, previousBit8, nextFibIndex, readBitId } = this;
+
+    let largestTreeIndex = 0;
```

Comment:
Should this variable be called `largestValue`? Nothing in here suggests it has to be `treeIndex`.